app/models/booking.rb

```diff
@@ -0,0 +1,4 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :meals
```

Comment:
should we need define the status in here:
enum :status, { confirmed: 0, pending: 1, canceled: 2 }